setup.py

```diff
@@ -47,7 +47,7 @@ def make_distribution(self):
     long_description=long_description,
     long_description_content_type="text/markdown",
     install_requires=[
-        "pandas==2.0.2",
+        "pandas>1,<2.1",
```

Comment:
Imagine a case when the user has pandas=1.3.5 and python 3.7. Then he wants to install modin. However, Modin only supports python>=3.8. Maybe we should remove pandas patch level pin starting 2.0?

Reply:
users on pandas 1.3.5 and python 3.7 can't use the latest version of modin anyway because modin now requires python >= 3.8. I think this is a separate issue. What do you suggest that we change? 

I suspect that even importing the latest modin code on python 3.7 would break because we make so many assumptions about what we can import from pandas. That was why backwards compatibility with python 3.6 was so much work. We should only consider backwards compatibility with python 3.7 if we know of a desire for it.

Reply:
The changes now are fine by me.